src/main/java/org/gridsuite/study/server/dto/SecurityAnalysisParametersInfos.java

```diff
@@ -0,0 +1,27 @@
+/**
+ Copyright (c) 2023, RTE (http://www.rte-france.com)
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.dto;
+
+import com.powsybl.security.SecurityAnalysisParameters;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Map;
+
+/**
+ * @author David Braquart <david.braquart@rte-france.com>
+ */
+@Getter
+@AllArgsConstructor
+@Builder
+public class SecurityAnalysisParametersInfos {
+
+    private SecurityAnalysisParameters parameters;
+
+    private Map<String, String> loadFlowSpecificParameters;
```

Comment:
why not using : List\<LoadFlowSpecificParameterInfos\> ?

Reply:
Yes I could have used that.
I choose the Map because this is what is used to create the extentions (in the compuation servers), and there is no need the declare/use LoadFlowSpecificParameterInfos in every computation server.

Do you think it is better/clearer to use List<LoadFlowSpecificParameterInfos> ?

Reply:
No, the change is not needed ...